Added Instructions

rishabh-vasudevan opened the pull request:
I have added the specifications and instructions that the user can follow to submit their voice samples

rishabh-vasudevan committed:
Added Instructions

Adding the specifications and instructions for the users to submit their voice samples

ctron commented on README.md:
```diff
@@ -1,3 +1,71 @@
+
+| __Phrase__ | "Hey Rhodney"   |
+| ------------- | ------------- |
+| __Number of Samples__  | 3  |
```
Maybe it would be good to clarify here if you want to have three files, or three samples in a single file.

ctron commented on README.md:
```diff
@@ -1,3 +1,71 @@
+## Step 1: Record the Audio
+
+### Steps to implement in audacity:
+1. Open audacity 
```
Where do I get "Audacity" from?

rishabh-vasudevan replied:
I will add the steps to download audacity

ctron commented on README.md:
```diff
@@ -1,3 +1,71 @@
+
+__Export the audio file in the form of a .wav file__:<br> 
+*Create a dedicated folder to store all recordings*
+1. After completing the recording 
```
```suggestion
1. After completing each recording 
```

ctron commented on README.md:
```diff
@@ -1,3 +1,71 @@
+1. After completing the recording 
+2. Go to File
+3. Export
+4. Export as *.wav*
```
Maybe note the "encoding" drop down? Which encoding should be selected.

ctron commented on README.md:
````diff
@@ -1,3 +1,71 @@
+
+__Linux__: 
+
+```zip -r [Name of the new folder.zip] [Name of the folder that contains the voice samples]```
````
Something seems off with the formatting from here on. I think you need to put the ``` in different lines.

ctron commented on README.md:
```diff
@@ -1,3 +1,71 @@
+
+### Upload the folder to the GitHub repository *“hey-rodney-dataset”* by creating an issue:
+
+- Open the url : https://github.com/rishabh-vasudevan/hey-rodney-dataset
```
Don't we use `https://github.com/drogue-iot/hey-rodney-dataset`?

rishabh-vasudevan replied:
copied the wrong link by mistake

ctron commented on README.md:
```diff
@@ -1,3 +1,71 @@
+### Upload the folder to the GitHub repository *“hey-rodney-dataset”* by creating an issue:
+
+- Open the url : https://github.com/rishabh-vasudevan/hey-rodney-dataset
+- Go to the issues tab on the top  
```
I think you could directly link to the issues page. Maybe even to the "New Issue" page.

rishabh-vasudevan replied:
yes that actually will be better

ctron commented on README.md:
```diff
@@ -1,3 +1,71 @@
+- Open the url : https://github.com/rishabh-vasudevan/hey-rodney-dataset
+- Go to the issues tab on the top  
+- Click on New Issue 
+- Make the title your name ( if you are comfortable sharing it ) or just say audio sample  
```
I don't think a real name is required.

ctron commented on README.md:
```diff
@@ -1,3 +1,71 @@
+- Click on New Issue 
+- Make the title your name ( if you are comfortable sharing it ) or just say audio sample  
+- Drag and drop the file to the place where it says to leave a comment  
+- You can also add your age, gender, nationality ( if you are comfortable with sharing it )
```
I think that is very personal information. In any case, there should be some kind of instruction on what and how (which form) to provide it.

rishabh-vasudevan replied:
I have added this only for the statistics. We can remove this part

ctron reviewed (changes requested):
Thanks for the PR!

We definitely need to mention that the audio samples will be licensed under the creative commons (CC0 aka Public Domain).

Regarding creating an issue, maybe a GitHub issue template would be helpful here.

rishabh-vasudevan committed:
Added Audacity installation steps and a few other things

rishabh-vasudevan committed:
removed name, age from steps to upload data on GitHub

Only left gender and first language in steps to upload data on GitHub. Added a note specifying the reason we are collecting this data.

rishabh-vasudevan committed:
Update issue templates

made a new submit voice sample issue template

ctron commented on README.md:
```diff
@@ -1,3 +1,82 @@
 
 This is a dataset of voice samples for our wake word detection of "Hey Rodney".
+
+All the audio samples will be licensed under the __Creative Commons Zero v1.0 Universal__ (CC0 aka Public Domain)
```
```suggestion
All the audio samples will be licensed under the __Creative Commons Zero v1.0 Universal__ (CC0 aka Public Domain). Only upload samples containing your own voice!
```

rishabh-vasudevan committed:
Update README.md

Co-authored-by: Jens Reimann <ctron@dentrassi.de>

rishabh-vasudevan committed:
Update issue templates

rishabh-vasudevan committed:
updated the issue url to the audio sample issue template
